pom.xml

```diff
@@ -37,12 +37,12 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <arrow.version>0.15.1</arrow.version>
```

Comment:
same upgrades for regular pom as fips pom. Commons-codec exclusions are already in place in this file.